server/athenian/api/controllers/team_controller.py

```diff
@@ -272,3 +276,66 @@ async def resync_teams(request: AthenianWebRequest, id: int) -> web.Response:
             teams, _ = await copy_teams_as_needed(
                 account, meta_ids, sdb_conn, request.mdb, request.cache)
     return await _list_loaded_teams(teams, account, meta_ids, request)
+
+
+async def team_exists(account_id: int, team_id: int, sdb_conn: DatabaseLike) -> bool:
+    """Whether the team exist in the given account."""
+    stmt = sa.select(1).where(sa.and_(Team.owner_id == account_id, Team.id == team_id))
+    return bool(await sdb_conn.fetch_val(stmt))
+
+
+async def get_recursive_team_members(
+    account_id: int, sdb_conn: DatabaseLike, root_team_id: Optional[int] = None,
+) -> Sequence[int]:
+    """Return the identifiers of the teams members, recursively collected in sub-teams.
+
+    If `root_team_id` is specified only that team and the recursively contained teams will be
+    considered, else all account teams will be considered.
+    If `root_team_id` is not None TeamNotFoundError will be raised if team is not found.
+
+    """
+    team_rows = await _recursive_teams_fetch(account_id, sdb_conn, (Team.members,), root_team_id)
+    if root_team_id is not None and not team_rows:
+        raise TeamNotFoundError(root_team_id)
+    all_members = set(chain.from_iterable(row[0] for row in team_rows))
+    return sorted(all_members)
+
+
+async def _recursive_teams_fetch(
+    account_id: int,
+    sdb_conn: DatabaseLike,
+    select_entities: Sequence[InstrumentedAttribute] = (Team.id,),
+    root_team_id: Optional[int] = None,
+    max_depth: Optional[int] = None,
+) -> Sequence[Mapping[Union[int, str], Any]]:
+    """Return the recursively collected list of teams for the account."""
+    # a recursive CTE is used to link children teams and track depth
```

Comment:
As we discussed, let's change to the simple team members fetch or loading the org members if id = 0